expression/builtin_arithmetic_vec_test.go

```diff
@@ -184,6 +185,55 @@ func (s *testEvaluatorSuite) TestVectorizedBuiltinArithmeticFunc(c *C) {
 	testVectorizedBuiltinFunc(c, vecBuiltinArithmeticCases)
 }
 
+func (s *testEvaluatorSuite) TestVectorizedErrOverflow(c *C) {
+	testCases := []vecExprBenchCase{
```

Comment:
actually i'm not sure if `vecExprBenchCase` is the correct structure for the error message test (e.g. `TestSleepVectorized` doesn't use `vecExprBenchCase` either)

Reply:
I think you are right, as using `vecExprBenchCase` has to introduce other variables, so I rewrite the test case like a lite `testVectorizedBuiltinFunc`